### PR TITLE
Use type option in read response

### PR DESCRIPTION
**Pull request by bobholt**
Doesn't infer the type from the input

### CODE REVIEW

File: modules/formatter-jsonapi/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function (input, opts) {
   // more than once to the top level `linked` key.
   var linkedResourceIndex = {};
   // get the underlying model type so we know what the primary resource is
```
Comment (tkellen) [inline modules/formatter-jsonapi/index.js]: this comment is now invalid

Comment (bobholt) [inline modules/formatter-jsonapi/index.js]: Good to just remove "underlying," or nuke it?

Comment (tkellen) [inline modules/formatter-jsonapi/index.js]: nuke it!

Comment (tkellen) [inline modules/formatter-jsonapi/index.js]: actually, remove the "underlying" and this is :+1:

**Comment by bobholt** [comment]
Oops, forgot to merge. This will get clobbered by #23.

**Pull request closed without merging**